handlers/jobs.go

```diff
@@ -116,17 +119,35 @@ func jobStatusToString(status *batchv1.JobStatus) string {
 	return "Unknown"
 }
 
-func createK8sJob(inputData string, accessToken string, userName string) (*JobInfo, error) {
+func createK8sJob(inputData string, accessToken string, pelicanCreds PelicanCreds, peregrineCreds PeregrineCreds, userName string) (*JobInfo, error) {
```

Comment:
Do we always need PelicanCreds and PeregrineCreds for creating a k8s job

Reply:
Right now there are only two job types exist, for `pelican` it does require both. I want to address more granular configuration (so, all configuration will go into the manifest) next sprint.